v_5_0_0/files/manifests/k8s-api-healthz.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       command:
         - /k8s-api-healthz
         - --api-endpoint="https://$(HOST_IP):443/healthz"
-      image: quay.io/giantswarm/k8s-api-healthz:1c0cdf1ed5ee18fdf59063ecdd84bf3787f80fac
+      image:  {{ .RegistryDomain }}/giantswarm/k8s-api-healthz:1c0cdf1ed5ee18fdf59063ecdd84bf3787f80fac
```

Comment:
we need to bump the version as only the latest tag is in china

Reply:
`0999549a4c334b646288d08bd2c781c6aae2e12f` should be the last tag and should be also in china

Reply:
@calvix Thanks. Done and confirmed here.  https://circleci.com/gh/giantswarm/k8s-api-healthz/40